conf/v1.routes

```diff
@@ -1,2 +1,5 @@
 PUT        /:nino/:businessId/period/:submissionId           v1.controllers.AmendForeignPropertyController.handleRequest(nino: String, businessId: String, submissionId: String)
+
+PUT        /:nino/:businessId/period/:submissionId          v1.controllers.AmendForeignPropertyController.handleRequest(nino: String, businessId: String, submissionId: String)
 GET        /:nino/:businessId/period/:submissionId          v1.controllers.RetrieveForeignPropertyController.handleRequest(nino: String, businessId: String, submissionId: String)
+POST       /:nino/:businessId/period                        v1.controllers.CreateForeignPropertyController.handleRequest(nino: String, businessId: String)
```

Comment:
Can this be moved above the rest? So the URLs are kind of in the same order as in the RAML